frontend/packages/topology/src/utils/__tests__/application-utils.spec.ts

```diff
@@ -172,6 +175,27 @@ describe('ApplicationUtils ', () => {
       .catch((err) => fail(err));
   });
 
+  it('Should delete all the specific models related to kamelet binding', async (done) => {
```

Comment:
can we add similar test for the new KafkaSink as well?  I see you have added test data `sampleSourceKafkaSink` but no see any tests related to it 🤔

Reply:
fixed